src/Tests/SettingsTest.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
```

Comment:
Yes I agree. Ideally, I would like to have the units extensively explored, and then just a poke at functional level to ensure the chain still flows. This one here simply sneaked through my todo list.